frontend/functions/wordpress/gravityForms/insertGfFormEntry.js

```diff
@@ -20,7 +20,7 @@ export default async function insertGfFormEntry(formId, fieldValues) {
   }
 
   // Convert File objects to be accepted on WP side.
-  fieldValues = fieldValues.map((field) => {
+  const newFieldValues = fieldValues.map((field) => {
```

Comment:
Hoe does creating a new variable fix this? It is still broken. by the time it gets to the `api/gravityforms` The FileList Is converted to a string